obp-commons/src/main/scala/com/openbankproject/commons/dto/JsonsTransfer.scala

```diff
@@ -478,7 +479,7 @@ case class OutBoundCreateCounterparty(outboundAdapterCallContext: OutboundAdapte
 
 case class InBoundCreateCounterparty(inboundAdapterCallContext: InboundAdapterCallContext, status: Status, data: CounterpartyTraitCommons) extends InBoundTrait[CounterpartyTraitCommons]
 
-case class OutBoundGetTransactionRequests210(outboundAdapterCallContext: OutboundAdapterCallContext, initiator : User, fromAccount : BankAccount) extends TopicTrait
+case class OutBoundGetTransactionRequests210(outboundAdapterCallContext: OutboundAdapterCallContext, @ignore initiator : User, fromAccount : BankAccountCommons) extends TopicTrait
```

Comment:
All follow BankAccountCommons should rollback to BankAccount, and @ignore should set in BankAccount.